ietf-document/draft-0rtt-bdp/draft-kuhn-quic-0rtt-bdp-07.xml

```diff
@@ -72,56 +72,214 @@
+			faster startup of flows and better buffer management.
+			The BDP metadata extension is proposed and candidate
+			parameters are discussed in <xref
+			target="sec:bdp_metadata"></xref>.</t>
 		</section>
 
 		<section anchor="sec:rationale" title="Motivation">
-			<t>Egress acceleration like 0-RTT "initial_max_data" alone does not improve the time-to-service. While "initial_max_data" improves the egress time to server, the BDP metadata extension aims at improving ingress traffic delivery.</t>
-			<!-- <t>QUIC encryption of transport headers prevents the adding of Performance-enhancing proxy (PEP). The BDP metadata extension may be a substitute to PEP proxy <xref target="RFC2488"></xref> <xref target="RFC3135"></xref> when time-to-service improvement requires acceleration of the refilling of client application buffers.</t> -->
-			<!-- <t>There are reconnection situation where the time-to-service depends only on server data arrival because the service logic does not requires any additional information from the client.</t> -->
-			<t>Currently each side has its proprietary solution to measure and to store path characteristics. Recalling the parameters of a previous connection would:<list style="symbols">
-					<t>allow server and client to immediately recover parameters that do not need to be negociated or estimated again (minRTT, MTU, bottleneck capacity, etc.)</t>
-					<t>allow the client to prepare the right resources</t>
-					<t>allow server side adaptation to non-default path characteristics</t></list></t>
-			<!-- <t>Avoid peers to compute the same path characteristics and decide opposed strategies: When the BDP is high the server may decide to increase the data on the flight while a resource-limited client will decide, as the ingress is expected to be slow, to reduce the resources.</t> -->
-			<!-- <t>Having a symmetrical optimization reduces ossification. Having the server to share the path characteristics avoid duplicate works and allows resource-limited client to save resources like CPU, memory and power. Tune the default transport parameters of network paths which have path characteristics that increase the time-to-service.</t> -->
+			<t> Reducing the number of round-trips required to
+			start a connection is an important way to reduce setup
+			time and lower overall connection latency. 0-RTT
+			mechanisms that allow a client to get feed requests to
+			a server in the first RTT do not alone improve the
+			total time-to-service. The BDP extension described in
+			this document aims to improve traffic delivery by
+			allowing the connection to short cut slow RTT based
+			processes that grow connection parameters.</t>
+
+			<t>Currently each side has its proprietary solution to
+			measure and to store path characteristics. Recalling
+			the parameters of a previous connection would allow:
+			<list style="symbols">
+					<t>Server and client to
+					immediately recover measured
```

Comment:
I am not overly keen on "immediately recover" - perhaps something like "can re-initialise the state from previously..." or something like that?